apollo-relay/README.md

```diff
@@ -0,0 +1,77 @@
+# Apollo Telemetry
+Relay statistics from router to apollo studio ingress.
+
+## Authentication
+
+The server must be authenticated to submit messages to the Apollo ingress.
+Furthermore, the statistics must be submitted "to" a particular graph. In
+the existing products this is accomplished using [environment variables](https://www.apollographql.com/docs/federation/managed-federation/setup/#4-connect-the-gateway-to-studio).
+
+In the router, we have a configuration file which can be dynamically
+re-loaded, so it makes more sense to include this configuration here. There
+is a new optional section that looks like this:
+
+```
+graph:
+  key: <YOUR_GRAPH_API_KEY>
+  reference: <YOUR_GRAPH_ID>@<VARIANT>
+```
+
+## Design
+
+There are two main components:
+ - Apollo Telemetry
+ - Relay
+
+### Configuration
+
+The telemetry statistics are internally delivered via gRPC service to a relay
+which then buffers data before finally delivering statistics to the Apollo
+ingress. That relay can be internal, which is the default, or external.
+
+The relay is configured from a new optional configuration section which looks
+like this:
+
+```
+studio:
```

Comment:
We have been avoiding putting the name of our SaaS product into configuration directives because we changed it several times and updating configuration directives and other software that had the name was a real pain.  (We hopefully will not do this again, but we already did it *three times*!)  (There's also a bit of a philosophical question as to whether "Studio" is our entire SaaS product or whether it is merely the UI.) So configuration of usage reporting in Apollo Server now explicitly does not use the word "Studio" in any configuration keys, class names, etc. I think we should probably follow this pattern unless it's challenging to avoid it. (It's fine to refer to Studio in docs, but that's different from using it in tokens that have technical meanings.)

Reply:
Not at all challenging. I think I originally had it as "usage" or "agent" or something like that. I'll try and align it to whatever name we settle on for the thing currently known as "apollo-relay".

Reply:
Since we've gone with "spaceport" for the CFKNA "relay", the obvious name for this section is "spaceport". I'll go with that for the time being.